modin/pandas/test/test_io.py

```diff
@@ -580,12 +580,125 @@ def test_read_csv_col_handling(
             **kwargs,
         )
 
+    # General Parsing Configuration
+    @pytest.mark.parametrize("dtype", [None, True])
+    @pytest.mark.parametrize("engine", [None, "python", "c"])
+    @pytest.mark.parametrize(
+        "converters",
+        [
+            None,
+            {
+                "col1": lambda x: np.int64(x) * 10,
+                "col2": pd.to_datetime,
+                "col4": lambda x: x.replace(":", ";"),
+            },
+        ],
+    )
+    @pytest.mark.parametrize("skipfooter", [0, 10])
+    def test_read_csv_parsing_1(
+        self,
+        make_csv_file,
+        request,
+        dtype,
+        engine,
+        converters,
+        skipfooter,
+    ):
+        if request.config.getoption("--simulate-cloud").lower() != "off":
+            pytest.xfail(
+                "The reason of tests fail in `cloud` mode is unknown for now - issue #2340"
+            )
+        kwargs = {
+            "dtype": dtype,
+            "engine": engine,
+            "converters": converters,
+            "skipfooter": skipfooter,
+        }
+
+        unique_name = get_unique_filename()
+        make_csv_file(
```

Comment:
Is there a file with the same content created every time?

Reply:
Yes, see clause 2 of [comment](https://github.com/modin-project/modin/pull/2331#issuecomment-734177265).